### PR TITLE
Replace home page dummy NFT with campaign items

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/NFTAboutPage/CTASection.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { logTrackerEvent } from '~/util/EventLogger';
+
+import walletMixin from '~/mixins/wallet';
+
+export default {
+  name: 'NFTCTASection',
+  mixins: [walletMixin],
+  methods: {
+    handleClickCampaignPageButton() {
+      // Legacy event
```
Comment (AuroraHuang22) [inline src/components/NFTAboutPage/CTASection.vue:25]: Should add a `this.$emit('go-campaign')` here ?

File: src/components/NFTAboutPage/CTASection.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      logTrackerEvent(this, 'NFT', 'NFTGoToCampaign', 'NFTGoToCampaign', 1);
+    },
+    handleClickMyPageButton() {
+      // Legacy event
```
Comment (AuroraHuang22) [inline src/components/NFTAboutPage/CTASection.vue:29]: Should add a  `this.$emit('go-dashboard')` here ?

File: src/pages/campaign/writing-nft.vue [hunk 3]
```diff
@@ -124,5 +125,18 @@ export default {
       return LIKECOIN_NFT_CAMPAIGN_ITEMS;
     },
   },
+  mounted() {
+    const { hash } = this.$route;
+    if (hash) {
+      try {
+        const el = document.querySelector(hash);
```
Comment (williamchong) [inline src/pages/campaign/writing-nft.vue:132]: What are the expected value of hash? does it need a `#` prefix?

File: src/components/NFTAboutPage/index.vue [hunk 4]
```diff
@@ -156,50 +157,47 @@
...
-    handleClickCampaignPageButton() {
-      logTrackerEvent(this, 'NFT', 'NFTGoToCampaign', 'NFTGoToCampaign', 1);
+    handleClickTopCTACampaignButton() {
+      logTrackerEvent(this, 'NFTAboutPage', 'CTA(Top)', 'GoToCampaign', 1);
+    },
+    handleClickTopCTADashboardButton() {
+      logTrackerEvent(this, 'NFTAboutPage', 'CTA(Top)', 'GoToMyDashboard', 1);
+    },
+    handleClickBottomCTACampaignButton() {
+      logTrackerEvent(this, 'NFTAboutPage', 'CTA(Bottom)', 'GoToCampaign', 1);
```
Comment (williamchong) [inline src/components/NFTAboutPage/index.vue:180]: Name these as `ClickCTA`?

File: src/pages/campaign/writing-nft.vue (hunk 3 above)
Comment (nwingt) [reply src/pages/campaign/writing-nft.vue:132]: The hash is returned with `#` already

**Review by AuroraHuang22 — APPROVED** [review]

**Pull request merged**